couchbase/src/options.rs

```diff
@@ -172,24 +172,47 @@ impl RemoveOptions {
     }
 }
 
+// This maps to LCB_N1QL_CONSISTENCY enum in couchbase.h
+#[repr(u32)]
+#[derive(Debug, Clone, Copy)]
+pub enum ScanConsistency {
```

Comment:
To align with SDK 3, please name it QueryScanConsistency (since there are other scan consistency enums for analytics, views, etc.)

Reply:
On it!